custom-commands/smart-room-lite/skill/Smart-Room-Lite-Demo.json

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
Either use camel-case or dash style, but not both, so
SmartRoomLiteDemo.json, or
smart-room-lite-demo.json